numpy/lib/tests/test_histograms.py

```diff
@@ -538,12 +538,32 @@ def test_outlier(self):
         xcenter = np.linspace(-10, 10, 50)
         outlier_dataset = np.hstack((np.linspace(-110, -100, 5), xcenter))
 
-        outlier_resultdict = {'fd': 21, 'scott': 5, 'doane': 11}
+        outlier_resultdict = {'fd': 21, 'scott': 5, 'doane': 11, 'stone': 6}
 
         for estimator, numbins in outlier_resultdict.items():
             a, b = np.histogram(outlier_dataset, estimator)
             assert_equal(len(a), numbins)
 
+    def test_scott_vs_ise(self):
+        """Verify that Scott's rule and the ISE based method converges for normally distributed data"""
+
+        def nbins_ratio(seed, size):
+            np.random.seed(seed)
+            x = np.random.normal(loc=0, scale=2, size=size)
```

Comment:
Please use an explicit random state object, e.g.,
```
rng = np.random.RandomState(seed)
x = rng.normal(...)
```

This ensures that this test is also thread safe.